moab/meta.yaml

```diff
@@ -0,0 +1,17 @@
+package:
+  name: moab
+  version: 4.8.0
+
+source:
+  fn: moab.tar.gz
+  url: ftp://ftp.mcs.anl.gov/pub/fathom/moab-4.8.0.tar.gz
```

Comment:
@scopatz any chance this could be changed to do a git clone and then checkout, historically the developers of MOAB/CGM have named their tarballs poorly and often contained the wrong versions corresponding to their name. I typically tell people to always fetch from git where possible because of this, perhaps I'm too cautious?